bot/exts/utilities/githubinfo.py

```diff
@@ -192,7 +195,7 @@ async def on_message(self, message: discord.Message) -> None:
 
             log.trace(f"Found {issues = }")
             # Remove duplicates
-            issues = set(issues)
+            issues = list(unique_everseen(issues))
```

Comment:
I'm not sure adding more_itertools just for this is worth it, when the below uses only built-ins, and is faster.

```suggestion
            issues = list(dict.fromkeys(issues))
```